app/src/main/res/values/strings.xml

```diff
@@ -47,5 +47,6 @@
     <string name="title_activity_learning_mode">Tryb Nauki</string>
     <string name="learning_mede_stop">X</string>
     <string name="alert_message_tryagain">Napisz go ponownie.</string>
+    <string name="learningmode_emptybase">Nie masz czego się jeszcze uczyć, najpierw musisz dodać słowka.</string>
```

Comment:
może "Zanim zaczniesz naukę, dodaj słówka do listy" "Before you start learning, add some new words to the list"